.travis.yml

```diff
@@ -2,7 +2,8 @@ language: node_js
 sudo: false
 addons:
   code_climate:
-    repo_token: 36766266e0f1834997de90e358ea07873eccb07006f9efb7369c35823024481c
+    repo_token:
```

Comment:
2 notes:

- the plaintext token will still be in the git history
- We don't consider them to be private data